packages/EasySchedule/src/ScheduleRunner.php

```diff
@@ -4,26 +4,21 @@
 
 namespace EonX\EasySchedule;
 
+use EonX\EasyEventDispatcher\Interfaces\EventDispatcherInterface;
 use EonX\EasyLock\Interfaces\LockServiceInterface;
+use EonX\EasySchedule\Events\CommandExecutedEvent;
 use EonX\EasySchedule\Interfaces\ScheduleInterface;
 use EonX\EasySchedule\Interfaces\ScheduleRunnerInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 final class ScheduleRunner implements ScheduleRunnerInterface
 {
-    /**
-     * @var \EonX\EasyLock\Interfaces\LockServiceInterface
-     */
-    private $lockService;
+    private bool $ran = false;
 
-    /**
-     * @var bool
-     */
-    private $ran = false;
-
-    public function __construct(LockServiceInterface $lockService)
-    {
-        $this->lockService = $lockService;
+    public function __construct(
+        private EventDispatcherInterface $eventDispatcher,
```

Comment:
Those arguments can even be marked as `readonly`

Reply:
Done